docs/public/tests.md

```diff
@@ -8,7 +8,6 @@ To configure tests locally you need to perform several steps:
    `.\venv\Scripts\activate`):
    ```bash
    python -m pip install -r requirements_qa.txt
```

Comment:
revert

Reply:
Unfortunately, I haven't understood what exactly you mean by that. I didn't do anything in tests.md file.

Reply:
But I see (and you, I think) that the line has been changed. Please add ``` as it was

Reply:
done